_docs_integrate/update-attributes-by-succession.md

```diff
@@ -110,6 +112,8 @@ Equally to IdentityAttributes, again only the `owner` can [succeed a Relationshi
 Doing so, a new version of the own shared RelationshipAttribute will be created at your side.
 The `succeeds` property of this LocalAttribute links to the old version, whose `succeededBy` property in turn is updated to the `id` of the newly created successor.
 Then, a Notification is sent automatically to the peer, which triggers the creation of a new peer shared RelationshipAttribute at their side, such that their LocalAttribute versions replicate the succession chain at your side.
+Please note that the Notification is queued if the [Relationship is currently terminated]({% link _docs_integrate/terminate-relationships.md %}#terminate-an-active-relationship).
+It can then only be received and processed once the [Relationship has been reactivated]({% link _docs_integrate/terminate-relationships.md %}#reactivate-a-terminated-relationship).
```

Comment:
In both the "Delete Attributes" and "Update Attributes by succession" scenarios, I found two places where the sending of Notifications is mentioned and I made a corresponding comment about queuing of Notifications in case of terminated Relationships. 

@Milena-Czierlinski Since you were more involved in implementing the corresponding features, please check whether queuing Notifications makes sense in all cases or whether we have or would like more restrictive validation in some cases.

Reply:
In my opinion, queuing Messages with Notifications makes sense for all Notifications we currently have, i.e. no further validation is needed.